typescript/src/apis/SessionApi.ts

```diff
@@ -139,6 +149,69 @@ export class SessionApi extends runtime.BaseAPI {
         await this.logoutHandlerRaw(initOverrides);
     }
 
+    /**
+     * Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.  # Errors  This call fails if Open ID Connect is disabled, misconfigured or the Id Provider is unreachable. 
```

Comment:
why are all comments doubled?

Reply:
We have it as our Rust-Doc and in the Utoipa-Macro, this means the generated JSON looks like this:

```json
"/oidcInit": {
      "post": {
        "tags": [
          "Session"
        ],
        "summary": "Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.",
        "description": "Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.\n\n# Errors\n\nThis call fails if Open ID Connect is disabled, misconfigured or the Id Provider is unreachable.\n",
        "operationId": "oidc_init",
        "responses": {
          "200": {
            "description": "",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/AuthCodeRequestURL"
                },
                "example": {
                  "url": "http://someissuer.com/authorize?client_id=someclient&redirect_uri=someuri&response_type=code&scope=somescope&state=somestate&nonce=somenonce&codechallenge=somechallenge&code_challenge_method=S256"
                }
              }
            }
          }
        }
      }
    },
```